core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -282,6 +282,40 @@ class MasterSuite extends SparkFunSuite
     }
   }
 
+  test("SPARK-24621 https urls when ssl enabled") {
```

Comment:
nit: we usually put a `:` after the JIRA number, eg. `SPARK-24621:`

Reply:
done